assets/css/style.css

```diff
@@ -1,6 +1,6 @@
 body {
   font-family: 'Roboto', sans-serif;
-  margin: 21px 27px;
```

Comment:
Let's stick with 0px or no margin set at the body level. We can set margins on individual elements.